### PR TITLE
Release: npm@6.4.0

### PR DESCRIPTION
**Pull request by zkat**
[Rendered CHANGELOG](https://github.com/npm/cli/blob/release-next/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,79 @@
...
+  ([@isaacs](https://github.com/isaacs))
+* [`50df1bf69`](https://github.com/npm/cli/commit/50df1bf691e205b9f13e0fff0d51a68772c40561)
+  `hosted-git-info@2.7.1`
+  ([@iarna](https://github.com/iarna))
+  ([@Erveon](https://github.com/Erveon))
+  ([@huochunpeng](https://github.com/huochunpeng))
+
+### DOCUMENTATION
+
+* [`c3ab25f3f`](https://github.com/npm/cli/commit/c3ab25f3f54038a813f765845a72ee9f9d836d7d)
```
Comment (iarna) [inline CHANGELOG.md]: This is really just a chore and doesn't need to be in the changelog.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,79 @@
+## v6.4.0 (2018-09-08):
+
+### NEW FEATURES
+
+* [`6e9f04b0b`](https://github.com/npm/cli/commit/6e9f04b0baed007169d4e0c341f097cf133debf7)
```
Comment (iarna) [inline CHANGELOG.md:5]: wow the implementation for that ended up being waaay better than it originally was

**Review by iarna — APPROVED** [review]

**Comment by iarna** [comment]
🐑

**Comment by brody2consult** [comment]
They just published node-gyp@3.8.0 which now uses request@^2.8.7 to resolve the `npm audit` issues, hope to see this package and other npm dependencies updated in turn.

_ref: <https://github.com/nodejs/node-gyp/pull/1521>_

File: CHANGELOG.md (hunk 2 above)
Comment (zkat) [reply CHANGELOG.md:5]: I agreeee

**Comment by zkat** [comment]
@brodybits nice! Thanks for pointing that out! And thanks @rvagg for getting the release out! 🎉  I've updated our dep updates and the changelog. 👍

**Comment by zkat** [comment]
`npm@6.4.0-next.0` has been [tagged and released](https://npm.community/t/release-npm-6-4-0-next-0/1289). This PR will stay open until `npm@6.4.0` goes live. 👍

**Pull request merged**